Move testdriver-vendor.js to LayoutTests/resources/

gsnedders opened the pull request:
#### 8fad6f4a06919723cce3142ae330cc1462f26fcb
<pre>
Move testdriver-vendor.js to LayoutTests/resources/
<a href="https://bugs.webkit.org/show_bug.cgi?id=292313">https://bugs.webkit.org/show_bug.cgi?id=292313</a>

Reviewed by Tim Nguyen.

* LayoutTests/fast/css/resize-above-min-size-and-below-initial-size.html:
* LayoutTests/fast/css/resize-below-min-intrinsic-size-large-scrollbars.html:
* LayoutTests/fast/css/resize-below-min-intrinsic-size.html:
* LayoutTests/fast/css/resize-below-min-size-zoomed.html:
* LayoutTests/fast/css/resize-below-min-size.html:
* LayoutTests/fast/css/resize-orthogonal-containing-block.html:
* LayoutTests/fast/css/resize-textarea-align-content.html:
* LayoutTests/fullscreen/full-screen-iframe-with-max-width-height.html:
* LayoutTests/fullscreen/fullscreen-enclosing-iframe-size.html:
* LayoutTests/fullscreen/fullscreen-iframe-navigation.html:
* LayoutTests/fullscreen/iframe-fullscreen-system-exit.html:
Adjust path to testdriver-vendor.js

* LayoutTests/imported/w3c/resources/config.json:
Add our testdriver-vendor.js to the server config; remove the
already-deleted testharness.css.

* LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js:
Replace with (empty) upstream file.

* LayoutTests/resources/testdriver-vendor.js:
Moved from LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js.

Canonical link: <a href="https://commits.webkit.org/296018@main">https://commits.webkit.org/296018@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/71b7a6148a1adb3bb435176be62ecf85b8411e52

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/106552 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/26303 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/16700 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/111756 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/57148 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/26973 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/34804 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/80910 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/109556 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/21387 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/96143 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/61245 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/20839 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/14247 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/56591 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/90726 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/14280 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/114618 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/33690 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/24827 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/89982 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/34054 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/92374 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/89691 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/22959 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/34596 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/12407 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/29303 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/33615 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/33361 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/36714 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/34960 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/resources/testdriver-vendor.js:
strange that github doesn't reflect this as a file move

gsnedders replied:
Matches `git`'s behaviour, with the other file being so totally changed.

nt1m reviewed (approved):
LGTM since it doesn't appear to break infra.

gsnedders committed:
Move testdriver-vendor.js to LayoutTests/resources/
https://bugs.webkit.org/show_bug.cgi?id=292313

Reviewed by Tim Nguyen.

* LayoutTests/fast/css/resize-above-min-size-and-below-initial-size.html:
* LayoutTests/fast/css/resize-below-min-intrinsic-size-large-scrollbars.html:
* LayoutTests/fast/css/resize-below-min-intrinsic-size.html:
* LayoutTests/fast/css/resize-below-min-size-zoomed.html:
* LayoutTests/fast/css/resize-below-min-size.html:
* LayoutTests/fast/css/resize-orthogonal-containing-block.html:
* LayoutTests/fast/css/resize-textarea-align-content.html:
* LayoutTests/fullscreen/full-screen-iframe-with-max-width-height.html:
* LayoutTests/fullscreen/fullscreen-enclosing-iframe-size.html:
* LayoutTests/fullscreen/fullscreen-iframe-navigation.html:
* LayoutTests/fullscreen/iframe-fullscreen-system-exit.html:
Adjust path to testdriver-vendor.js

* LayoutTests/imported/w3c/resources/config.json:
Add our testdriver-vendor.js to the server config; remove the
already-deleted testharness.css.

* LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js:
Replace with (empty) upstream file.

* LayoutTests/resources/testdriver-vendor.js:
Moved from LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js.

Canonical link: https://commits.webkit.org/296018@main
